docs/home/whats-new-2-15.md

```diff
@@ -1,94 +1,103 @@
-# Run:ai version 2.15 - September 28, 2023
+# Run:ai version 2.15 - October 28, 2023
 
 ## New Features
 
 <!-- RUN-10221/RUN-10426 Projects V2 - User will be able to export a CSV report - NEW FEATURE -->
-* Added new functionality to download a report all the data in the table for the following:
-    * Projects
-    * Departments
-    * Nodes
-    * Node pools
-    * Consumption report
+* Added the ability to download a CSV file from all features that contain a table. Downloading a CSV can provide a snapshot of the feature history over the course of time, and help with compliance tracking. All the columns that are selected (displayed) in the table will be downloaded to the file.
 
-Use the *More* menu to download the CSV file. Select from the *Columns* menu to add more columns to the CSV file.
+<!-- RUN-7495/RUN11388 Support PSA / SCCs V2 \(security mechanism for pods on K8S/OCP\)-->
+* Added support for `restricted` policy for [Pod Security Admission](https://kubernetes.io/docs/concepts/security/pod-security-admission/){target=_blank} (PSA) on OpenShift only. For more information, see [Pod security admission](../admin/runai-setup/cluster-setup/cluster-prerequisites.md#pod-security-admission).
 
-## Improvements
+<!-- ADDLINK RUN-10241/RUN-12872 - New Workloads view -->
+* Added a new button to the *Jobs* feature to switch the view to *Workloads*. *Workloads* is a new view for jobs that are running in the platform. The *Workloads* view provides a more advanced UI than the previous *Jobs* UI. The new table format provides:
 
-<!-- TODO RUN-9943/RUN-12176 Nodes - reflect the correct status of the node - add to nodes page the table from the TW ticket -->
-* Improved the readability of the node table to include a more detailed status and its description. For more information, see [Page here](page here)
```

Comment:
it is more than readability. the status has become more helpful for explainability (internal term). @lavianalon , please review and rephrase focused on customer value (which is more than only UX)

Reply:
O...this one is the deleted one I will review the new one